packages/opentelemetry-plugin-grpc-js/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenTelemetry @grpc/grpc-js Instrumentation for Node.js
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`@grpc/grpc-js`](https://grpc.io/blog/grpc-js-1.0/). Currently, version [`1.x`](https://www.npmjs.com/package/@grpc/grpc-js?activeTab=versions) of `@grpc/grpc-js` is supported.
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-node) package.
+
+## Installation
+
+```
+npm install --save @opentelemetry/plugin-grpc-js
+```
+
+## Usage
+
+OpenTelemetry gRPC Instrumentation allows the user to automatically collect trace data and export them to the backend of choice, to give observability to distributed systems when working with [gRPC](https://www.npmjs.com/package/@grpc/grpc-js).
+
+To load a specific plugin (**gRPC** in this case), specify it in the Node Tracer's configuration.
+```javascript
+const { NodeTracerProvider } = require('@opentelemetry/node');
+
+const provider = new NodeTracerProvider({
+  plugins: {
+    grpcJs: {
+      enabled: true,
+      // You may use a package name or absolute path to the file.
+      path: '@opentelemetry/plugin-grpc-js',
+    }
+  }
```

Comment:
Might be good to share existing `grpc` config name here... Would require allowing `path` to be an array.

Reply:
Impossible because of https://github.com/open-telemetry/opentelemetry-js/pull/1148#discussion_r436916855